Python/Setup/swix/swix.generate.targets

```diff
@@ -49,10 +49,6 @@ package name=$([System.IO.Path]::GetFileNameWithoutExtension($(OutputName))) ver
 <WriteLinesToFile File="$(SwrFile).tmp" Lines="vs.dependencies
 $(_Dependencies)
 " Overwrite="no" Encoding="ASCII" Condition="$(_Dependencies) != ''" />
-
-    <ItemGroup>
-      <Package Include="$(SwrFile)" />
-    </ItemGroup>
   </Target>
 
   <Target Name="_GenerateSwrFromSources" Condition="@(SwrSource) != ''">
```

Comment:
Or you could just build a classic vsix and all it takes is a single line to reference it :wink: